arangod/Cluster/ClusterInfo.cpp

```diff
@@ -1660,6 +1660,10 @@ int ClusterInfo::createCollectionCoordinator(std::string const& databaseName,
         events::CreateCollection(name, TRI_ERROR_CLUSTER_TIMEOUT);
         return setErrormsg(TRI_ERROR_CLUSTER_TIMEOUT, errorMsg);
       }
+      
+      if (application_features::ApplicationServer::isStopping()) {
```

Comment:
I think we have to create the `events::CreateCollection` with a shutdown error here as well, to leave the auditing feature watertight.